src/Service/Context/PageContext.php

```diff
@@ -94,7 +95,9 @@ public function __construct(ConfigFactoryInterface $config_factory, EntityTypeMa
    */
   private function setPageContextCredential($credential_settings) {
     foreach (SELF::$CREDENTIAL_MAPPING as $credential_key => $tag_name) {
-      $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      if (isset($credential_settings[$credential_key]) {
+        $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      }
```

Comment:
That's a good thought. 

On a semi-separated concern, does the JS mind if empty meta tags are simply skipped and not dropped on the page at all? If yes, should it?